template/src/features/counter/counterSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const slice = createSlice({
+  name: 'counter',
+  initialState: {
+    value: 0,
+  },
+  reducers: {
+    increment: state => {
+      state.value += 1;
+    },
+    decrement: state => {
```

Comment:
I'd like to see one other case handled here that actually shows using the `action` parameter as well.  Maybe have an `incrementByAmount` case, and allow the user to enter a number in the UI?